### PR TITLE
Resolved Add Electro and Dubstep articles to the Sound section #1391

### PR DESCRIPTION
**Pull request by smriad**

### CODE REVIEW

**Comment by BaseMax** [comment]
Hi my friend,
Good job. john is now sleeping, he will review/merge soon...

**Review by BaseMax — DISMISSED** [review]
LGTM.

**Comment by smriad** [comment]
> Hi my friend,
> Good job. john is now sleeping, he will review/merge soon...

Okay. Thanks

File: doc/source/dubstep.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+The Every Noise at Once platform provides an opportunity for people to get to know better the names of the creators who have ever been in the genre and thus the peculiarities of it.
+<https://everynoise.com/everynoise1d.cgi?root=classic%20dubstep&scope=all>
+
+## Sources:
+-MENNYEK KAPUI - Az elektroniks zene evtizede (The decade of electronic music)
+
+-Ishkur's Guide to Electronic Music
+
+-Every Noise at Once
```
Comment (jbampton) [inline doc/source/dubstep.md:31]:
```suggestion
- MENNYEK KAPUI - Az elektroniks zene evtizede (The decade of electronic music)

- Ishkur's Guide to Electronic Music

- Every Noise at Once
```

File: doc/source/electro.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+You can find more details about its formation and development on Ishkur's Guide to Electronic Music, and you can listen to acid 
+house music right away.
+<https://music.ishkur.com/?query=Electro>#
+
+## Sources: 
+ - MENNYEK KAPUI - Az elektroniks zene évtizede (The decade of electronic music)
+ - Wikipedia 
+ - Ishkur's Guide to Electronic Music
+
+## About the author:
```
Comment (jbampton) [inline doc/source/electro.md:40]:
```suggestion
## About the author:

```

File: doc/source/index.rst [hunk 3]
```diff
@@ -84,9 +84,11 @@ Welcome to slurp's documentation!
    acid-trance.md
    ambient.md
    big-beat.md
+   dubstep.md
    goa.md
    hardcore.md
    house.md
+   electro.md
```
Comment (jbampton) [inline doc/source/index.rst:91]:
```suggestion
   electro.md
   goa.md
   hardcore.md
   house.md
```

**Comment by ayushrana182** [comment]
LGTM!

**Review by jbampton — APPROVED** [review]

**Pull request merged**